src/bci_build/package/postfix/README.md.j2

```diff
@@ -91,16 +91,16 @@ Mails for virtual domains and users are stored in Maildir format in
 `/var/spool/vmail` inside the container. To not loose the mail, this directory
 should be a persistence volume.
 `VMAIL_UID` specifies the UID and GID which owns all files below this
-directory. This makes sure, that the container is using the same UID/GID as
-the Container Host OS for the files and not regular users are owning this
-files and can read and modify them. The default UID/GID is `5000`.
+directory. This ensures that the container is using the same UID/GID as
+the container host OS for the files and no regular users own these
+files and can read or modify them. The default UID/GID is `5000`.
 
-There are three ways to provide the data for virtual domains and users, via
-environment variables, files or LDAP.
+There are three ways to provide the data for virtual domains and users:
+via environment variables, files, or LDAP.
```

Comment:
in the SUSE doc guide, the `,` is often omitted in such cases.

Reply:
there is no guidance on oxford comma or not in the technical writing guide. personally I prefer it but I can remove it also. not really relevant here.